### PR TITLE
feat: Refactor <Label>

### PR DESCRIPTION
**Pull request by sebald**
- no variants anymore
- no disabled (not possible anyway)
- improve a11y

Ref #1889

### CODE REVIEW

**Comment by sebald** [comment]
No changeset since it is not publicly available yet.

File: packages/components/src/Field-Next/Label.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+// TODO: But this back into the root, when we realease all the new fields!
```
Comment (ti10le) [inline packages/components/src/Field-Next/Label.tsx:1]: *Put 
*release
😀

File: packages/components/src/Field-Next/Label.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      as={as}
+      variant="label"
+      __baseCSS={{ display: 'flex', alignItems: 'center', gap: 4 }}
+    >
+      {children}
+      {/*
+       * aria-required is set on the field and will already be announced,
+       * so we don't need to add it here.
+       */}
+      {required && <Required role="img" size={16} />}
```
Comment (ti10le) [inline packages/components/src/Field-Next/Label.tsx:48]: Just for m information: why role img?

**Review by ti10le — APPROVED** [review]

**Comment by ti10le** [comment]
@sebald questions hard ignored 😂

**Comment by sebald** [comment]
@ti10le don't mark it as approved 😝

File: packages/components/src/Field-Next/Label.tsx (hunk 2 above)
Comment (sebald) [reply packages/components/src/Field-Next/Label.tsx:48]: https://lmgtfy.app/?q=role+img 🤪

**Comment by ti10le** [comment]
That was intentional, because it can be gemerged, just thought I still get an answer 😀

**Pull request merged**